src/hobbits-plugins/displays/Spectrogram/spectrogramcontrols.cpp

```diff
@@ -32,22 +32,36 @@ SpectrogramControls::SpectrogramControls(QSharedPointer<ParameterDelegate> deleg
     m_paramHelper->addComboBoxParameter("data_type", ui->cb_dataType);
 
     m_paramHelper->addSliderIntParameter("sensitivity", ui->hs_sensitivity);
-    m_paramHelper->addSpinBoxIntParameter("fft_size", ui->sb_fftSize);
+   
+    ui->cb_fftSize->addItem("32", (pow(2.0, 5.0)));
+    ui->cb_fftSize->addItem("64", (pow(2.0, 6.0)));
+    ui->cb_fftSize->addItem("128", (pow(2.0, 7.0)));
+    ui->cb_fftSize->addItem("256", (pow(2.0, 8.0)));
+    ui->cb_fftSize->addItem("512", (pow(2.0, 9.0)));
+    ui->cb_fftSize->addItem("1,024", (pow(2.0, 10.0)));
+    ui->cb_fftSize->addItem("2,048", (pow(2.0, 11.0)));
+    ui->cb_fftSize->addItem("4,096", (pow(2.0, 12.0)));
+    ui->cb_fftSize->addItem("8,192", (pow(2.0, 13.0)));
+    ui->cb_fftSize->addItem("16,384", (pow(2.0, 14.0)));
+    ui->cb_fftSize->addItem("32,768", (pow(2.0, 15.0)));
+    ui->cb_fftSize->addItem("65,536", (pow(2.0, 16.0)));
```

Comment:
good idea! i changed it to use left shifts. i believe it will be clear what the left shifts mean as the first argument of addItem has the number (just as a string).